ServerCore/Pages/Rooms/RoomList.cshtml.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+using ServerCore.ModelBases;
+
+namespace ServerCore.Pages.Rooms
+{
+    [Authorize(Policy = "IsEventAdmin")]
+    public class RoomListModel : EventSpecificPageModel
+    {
+        public RoomListModel(PuzzleServerContext serverContext, UserManager<IdentityUser> manager) : base(serverContext, manager)
+        {
+        }
+
+        public IList<Room> Room { get; set; }
```

Comment:
If fixing pluralization here and in the db doesn't cause a mess, it would be nice

Reply:
Seems to be ok this time, I'm not seeing whatever I was running into before. Done.